services/web/src/pages/profile/index.page.jsx

```diff
@@ -42,7 +42,12 @@ const Profile = () => {
   const onSubmit = ({ password }) => updateCurrent({ password }, {
     onSuccess: (data) => {
       queryClient.setQueryData(['currentUser'], data);
-      toast.success('Your password have been successfully updated.');
+      showNotification({
```

Comment:
let's create some helper/hook for mantine `showNotification`.
Previous API was more convenient
`toast.success('Your password have been successfully updated.');`